### ISSUE TITLE
pb comparing negative floats

### ISSUE THREAD

**Issue by wagnerf42** [mikedilger/float-cmp#8]
```
extern crate float_cmp;
use float_cmp::ApproxOrdUlps;
fn main() {
    let x: f64 = -1.0;
    let y: f64 = -2.0;
    println!("-1 {:?} -2", x.approx_cmp(&y, 2));
    let x: f64 = 1.0;
    let y: f64 = 2.0;
    println!("1 {:?} 2", x.approx_cmp(&y, 2));
}
```

-1 is less than -2

**Comment by mikedilger** [comment]
Yeah, ouch.  That's pretty bad.  It might be obvious by now that I don't use my own crate much.

I've got a fix, but I think I'll do some more extensive testing before I release again.  approx_cmp() should behave exactly like partial_cmp() whenever ulps=0 and partial_cmp() returns Some result.  But it doesn't.  So I'll be wanting to fix that as well.

**Comment by mikedilger** [comment]
fixed and released as 0.2.5
